packages/smoke-tests/page-objects/add-friend.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { BasePageObject } = require('@faltest/page-objects');
+
+class AddFriend extends BasePageObject {
+  constructor(host, ...args) {
+    super(...args);
+
+    this.host = host;
+  }
+
+  get addFriendButton() {
+    // element click intercepted: Element <a class="button button-xs" href="/add-friend">...</a> is not clickable at point (274, 76). Other element would receive the click: <a class="service-worker-update-notify alert alert-info has-shadow" href="/chat" style="z-index: 100;">...</a>
```

Comment:
is this a layering issue? 

I _have_ been thinking about moving the service-worker-update-notify prompt to a different location. It gets visually weird when it's _also_ overlayed on the pls enable notifications banner

Reply:
yep layering issue